collectors/freeipmi.plugin/metadata.yaml

```diff
@@ -42,36 +42,225 @@ modules:
     setup:
       prerequisites:
         list:
-          - title: Preliminary actions
+          - title: Install freeipmi.plugin
             description: |
-              If you have not previously used IPMI on your system, you will probably need to run the ipmimonitoring command as root to initialize IPMI settings so that the Netdata plugin works correctly. It should return information about available sensors on the system.
+              When using our official DEB/RPM packages, the FreeIPMI plugin is included in a separate package named `netdata-plugin-freeipmi` which needs to be manually installed using your system package manager. It is not installed automatically due to the large number of dependencies it requires.
 
-              In some distributions libipmimonitoring.pc is located in a non-standard directory, which can cause building the plugin to fail when building Netdata from source. In that case you should find the file and linking it to the standard pkg-config directory. Usually, running sudo ln -s /usr/lib/$(uname -m)-linux-gnu/pkgconfig/libipmimonitoring.pc/libipmimonitoring.pc /usr/lib/pkgconfig/libipmimonitoring.pc resolves this issue.
+              When using a static build of Netdata, the FreeIPMI plugin will be included and installed automatically, though you will still need to have FreeIPMI installed on your system to be able to use the plugin.
 
-              If you installed `netdata` using a package manager, it is also necessary to install the package `netdata-plugin-freeimpi`.
+              When using a local build of Netdata, you need to ensure that the FreeIPMI development packages (typically called `libipmimonitoring-dev`, `libipmimonitoring-devel`, or `freeipmi-devel`) are installed when building Netdata.
+          - title: Preliminary actions
+            description: |
+              If you have not previously used IPMI on your system, you will probably need to run the `ipmimonitoring` command as root
+              to initialize IPMI settings so that the Netdata plugin works correctly. It should return information about available sensors on the system.
       configuration:
         file:
           name: "netdata.conf"
-          section_name: '[plugin:freeipmi]'
-          description: "This is netdata main configuration file"
+          section_name: "[plugin:freeipmi]"
         options:
-          description: "This tool receives command line options that are visible when user run: `./usr/libexec/netdata/plugins.d/freeipmi.plugin --help`"
+          description: |
+            The configuration is set using command line options. 
+
```

Comment:
@ilyam8 you approved the PR, but we have this warning about `trailing-spaces`, are we going to ignore it?